src/serverdlgbase.ui

```diff
@@ -225,96 +217,84 @@
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
-        <layout class="QHBoxLayout" name="horizontalLayout">
-         <item>
-          <widget class="QLabel" name="lblLanguage">
+        <layout class="QGridLayout" name="gridLayout_2">
+         <item row="4" column="1">
+          <widget class="QCheckBox" name="chbStartOnOSStart">
            <property name="text">
-            <string extracomment="Consider add &quot;(Lang)&quot; as a suffix to ensure that users who selected the wrong language find the correct button">Language</string>
+            <string>Start Minimized on Windows Start</string>
            </property>
           </widget>
          </item>
-         <item>
-          <widget class="CLanguageComboBox" name="cbxLanguage">
-           <property name="sizePolicy">
-            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-             <horstretch>0</horstretch>
-             <verstretch>0</verstretch>
-            </sizepolicy>
-           </property>
-          </widget>
-         </item>
-        </layout>
-       </item>
-       <item>
-        <layout class="QHBoxLayout">
-         <item>
+         <item row="1" column="0">
           <widget class="QPushButton" name="pbtRecordingDir">
            <property name="text">
             <string>Recording Directory</string>
            </property>
           </widget>
          </item>
-         <item>
-          <widget class="QLineEdit" name="edtRecordingDir">
-           <property name="readOnly">
-            <bool>true</bool>
+         <item row="0" column="0">
+          <widget class="QLabel" name="lblLanguage">
+           <property name="text">
+            <string extracomment="Consider add &quot;(Lang)&quot; as a suffix to ensure that users who selected the wrong language find the correct button">Language</string>
```

Comment:
I think this comment is grammatically incorrect.

```suggestion
            <string extracomment="Consider adding &quot;(Lang)&quot; as a suffix to ensure that users who selected the wrong language find the correct button">Language</string>
```

Reply:
Yep.  Unchanged but good spot (it moved).

Reply:
Changed and added to the Client Settings dialog, too.